### PR TITLE
Initial commit for hail-vep-pipeline datasets yaml file

### PR DESCRIPTION
**Pull request by privojack**
*Description of changes:* 

Initial commit for `hail-vep-pipeline.yaml` file in `datasets` directory.  Supporting a recent submission to the Registry of Open Data by Privo.

Please let me know if there are any questions or concerns, thanks!

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/hail-vep-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Name: Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin
+Description: VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release includes The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.
+Documentation: https://privoit.github.io/hail-vep-pipeline/
+Contact: https://www.privoit.com/contact-us
+ManagedBy: "[Privo](https://www.privoit.com/)"
+UpdateFrequency: New packages are added as soon as they are available and confirmed to work with recent versions of Hail.
+Tags:
+  - genome-wide association study
```
Comment (chueatwork) [inline datasets/hail-vep-pipeline.yml:8]: 1. Add - aws-pds to tags
2. Revise genome-wide association study to genome wide association study

I added vep and loftee to tags.yaml preemptively

File: datasets/hail-vep-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - vep
+  - loftee
+License: [VEP](https://uswest.ensembl.org/info/about/publications.html) use is governed by the Apache 2.0 licenses, and [LOFTEE](https://github.com/konradjk/loftee/blob/master/LICENSE) use is governed by the MIT license.
+Resources:
+  - Description: VEP and LOFTEE data
+    ARN: arn:aws:s3:::hail-vep-pipeline
+    Region: us-east-1
+    Type: S3 Bucket
+DataAtWork:
+  Tools & Applications:
```
Comment (chueatwork) [inline datasets/hail-vep-pipeline.yml:20]: Add Tutorials field (doesn't have to be filled in, but the field has to be there)

Comment (privojack) [reply datasets/hail-vep-pipeline.yml:22]: Added

File: datasets/hail-vep-pipeline.yml (hunk 1 above)
Comment (privojack) [reply datasets/hail-vep-pipeline.yml]: Added aws-pds tag and updated to remove hyphen from genome wide association study

**Comment by privojack** [comment]
Thanks for the quick feedback, I've responded to those changes.

**Comment by chueatwork** [comment]
Thanks! This looks ready, but can change the file suffix to .yaml ? Idiosyncrasy of our code build and it's not giving me visibility into any other blockers.

**Comment by privojack** [comment]
Oh, whoops!  Sorry about that.  It's been updated to `.yaml`

**Comment by chueatwork** [comment]
@privojack can you add a pipe in front of the License text, as follows:
License: |
[your existing text here]

Thanks!

**Comment by privojack** [comment]
Hello, I believe I've formatted that properly now.  Thanks!  I've also removed the `.yml` file in the newest commit.

**Comment by chueatwork** [comment]
Hey @privojack we were running into an issue where I'd patched a fix upstream, but couldn't update your base branch. For the sake of efficiency I just generated a new PR (can see your text at datasets/hail-vep-pipeline) and am closing this one. Thanks for bearing with =)

**Pull request closed without merging**